OrbitCore/FunctionFramepointerValidatorTest.cpp

```diff
@@ -0,0 +1,50 @@
+#include "FunctionFramepointerValidator.h"
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <capstone/capstone.h>
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#define WITH_FP "\x55\x89\xE5\x83\xC0\x01\xE8\x77\x00\x00\x00\x89\xEC\x5D"
```

Comment:
Do these need to be `define`s? Can they just be constant?

Reply:
Thanks for spotting this. Please see above.